### PR TITLE
Release to Github workflow

### PR DESCRIPTION
**Pull request by fefi95**
# Issue: #128 

##  Description
This PR sets up GH actions to create a release on Github and upload the assets for Linux and MacOS versions of the library.

### CODE REVIEW

File: .github/workflows/release-gh.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Release to package to Github
```
Comment (CristhianMotoche) [inline .github/workflows/release-gh.yml:1]: typo?
```suggestion
name: Release package to Github
```

**Review by CristhianMotoche — APPROVED** [review]
Hey @fefi95 Looks pretty good. Could you please merge it and then test it with some tag (e.g. `v.test`? We can remove that tag and release later.

File: .github/workflows/release-gh.yml (hunk 1 above)
Comment (fefi95) [reply .github/workflows/release-gh.yml:1]: Yes! Thanks for catching that 😄

**Comment by fefi95** [comment]
@CristhianMotoche Thanks for taking a look! Yes, I'll test it with that tag.

**Pull request merged**